servicex/databinder_models.py

```diff
@@ -241,3 +257,11 @@ class ServiceXSpec(BaseModel):
     """
     Any reusable definitions that are needed for the transform request
     """
+
+    @field_validator("Sample", mode="after")
+    @classmethod
+    def validate_unique_sample(cls, v):
+        hash_set = set([i.hash for i in v])
```

Comment:
I think this can be just `{i.hash for i in v}`

Reply:
Not needed anymore